Plugins/Public/base_plugin/PlayerBase.cpp

```diff
@@ -768,3 +768,16 @@ float PlayerBase::SpaceObjDamaged(uint space_obj, uint attacking_space_obj, floa
 
 	return 0.0f;
 }
+
+// Reset shield to default strength
+void PlayerBase::ResetShieldStrength() {
+	CoreModule *mod;
```

Comment:
Shouldn't this just be a part of line 776 where you set `mod`? This feels like the old JavaScript variable hoisting we used to do back in the day.